app/models/post.rb

```diff
@@ -0,0 +1,22 @@
+class Post < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
+  has_many :likes
+  has_many :comments
+
+  after_create :increment_post_counter
+  after_destroy :decrement_post_counter
+
+  def increment_post_counter
+    puts 'Incrementing post counter'
+    user.increment!(:post_counter)
+  end
+
+  def decrement_post_counter
+    puts 'Decrementing post counter'
+    user.decrement!(:post_counter)
+  end
+
+  def recent_comments(limit)
+    comments.order(created_at: :desc).limit(limit)
+  end
```

Comment:
- [ ] Same here. As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project_v1_1.md#:~:text=for%20a%20user.-,A%20method%20which%20returns%20the%205%20most%20recent%20comments%20for%20a%20given%20post.,-Comments), Please make the limit 5 comments only.